### ISSUE TITLE
Fail Accessibility Insights Fast Pass

### ISSUE THREAD

**Issue by Bakar0** [react-component/tree#426]
You have this div at the top of the tree with role = tree and input children with no child role.
this coz to fail in the Accessibility Insights Fast Pass.

<div role="tree"><input tabindex="0" value="" style="width: 0px; height: 0px; display: flex; overflow: hidden; opacity: 0; border: 0px; padding: 0px; margin: 0px;"></div>
